.travis.rosinstall

```diff
@@ -1,6 +1,8 @@
 #- git:
-#    uri: https://github.com/ipab-slmc/eigenpy_catkin.git
+#    uri: https://github.com/stack-of-tasks/eigenpy.git
+#    version: devel
 #    local-name: eigenpy
 #- git:
-#    uri: https://github.com/ipab-slmc/pinocchio_catkin.git
+#    uri: https://github.com/stack-of-tasks/pinocchio.git
+#    version: devel
```

Comment:
Can you make these point to a commit hash instead of `devel`?
Devel tends to get broken at 11pm on Fridays.

Reply:
This is not being used right now (commented). When I activate it temporarily on the buildserver, I will fix it to the corresponding version. We are directly using the ROS buildfarm binaries (not yet synced, but sync is upcoming)